### DISCUSSION TITLE
Add the support ApplicationRecordTypeConfig metadata type

### DISCUSSION THREAD

**Discussion by 92SGupta** [scolladon/sfdx-git-delta#967] [General]
Hi @scolladon : Can you please add below to the supported metadata:
    {
        "directoryName": "ApplicationRecordTypeConfig",
        "inFolder": false,
        "metaFile": false,
        "suffix": "applicationRecordTypeConfig",
        "xmlName": "ApplicationRecordTypeConfig"
    },

**Comment by scolladon** [comment]
Hi @92SGupta !

Sure, I'll add it soon,
Stay tuned!
